<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATP</class>
 <widget class="QWidget" name="ATP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #242c39;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="Side_Logo_frame">
     <property name="styleSheet">
      <string notr="true">background-color: #242c39;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Get_label">
        <property name="styleSheet">
         <string notr="true">background-color:None;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
        </property>
        <property name="text">
         <string>BBT.Live Project</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">#groupBox_2 {
border-color: rgb(204, 204, 0);
border-width : 0.5px;
border-style:inset}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QPushButton" name="update">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: #242c39;</string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="finish">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: #242c39;</string>
           </property>
           <property name="text">
            <string>Perform</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_50">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>BBT.jpg</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_9">
        <property name="styleSheet">
         <string notr="true">#groupBox_9 {
border-color: rgb(204, 204, 0);
border-width : 0.5px;
border-style:inset}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <widget class="QLabel" name="SAT_label_2">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:None;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
           </property>
           <property name="text">
            <string>UI SHELLS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>97</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/Widget's ATP/getsat2.JPG</pixmap>
      </property>
     </widget>
     <zorder>label_19</zorder>
     <zorder>Get_label</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>label_50</zorder>
     <zorder>groupBox_9</zorder>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">background-color: #242c39;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true">#groupBox_3 {
border-color: rgb(204, 204, 0);
border-width : 0.5px;
border-style:inset}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_15">
           <property name="styleSheet">
            <string notr="true">#groupBox_15 {
border-color: rgb(204, 204, 0);
border-width : 0.5px;
border-style:inset}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_40">
            <item row="0" column="1">
             <widget class="QLabel" name="label_17">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>16</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:None;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
              </property>
              <property name="text">
               <string>Choose A Command</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" colspan="3">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="1" column="2">
               <widget class="QLineEdit" name="ASSN_2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="ASREV_2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ifconfig</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>rout -n (beta)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>arp -n</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>iptables -t mangle -nvL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>iptables -t nat -nvL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>clear</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Enter IP</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Enter Command</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_13">
           <property name="styleSheet">
            <string notr="true">#groupBox_13 {
border-color: rgb(204, 204, 0);
border-width : 0.5px;
border-style:inset}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="2">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>16</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:None;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
              </property>
              <property name="text">
               <string>LAN Switching</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <layout class="QGridLayout" name="gridLayout_18">
              <item row="0" column="3">
               <widget class="QRadioButton" name="netB">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:None;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
                </property>
                <property name="text">
                 <string>NET B</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="netA">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:None;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
                </property>
                <property name="text">
                 <string>NET A</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QGroupBox" name="groupBox_12">
           <property name="styleSheet">
            <string notr="true">#groupBox_12 {
border-color: rgb(204, 204, 0);
border-width : 0.5px;
border-style:inset}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_21">
            <item row="0" column="2">
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_21">
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>16</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:None;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color:#ffc100;</string>
              </property>
              <property name="text">
               <string>Performer</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="2" column="0" colspan="2">
               <layout class="QGridLayout" name="gridLayout_31">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_55">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>Performed By</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="MTESTER">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                  <item>
                   <property name="text">
                    <string/>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>User 1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>User 2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>User 3</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
